doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -180,9 +180,9 @@ Using the nRF54H20 DK with the |NCS| v2.7.0 requires the following:
 
       nrfutil self-upgrade
 
-#. Install the nRF Util ``device`` command to version 2.4.0 as follows::
+#. Install the nRF Util ``device`` command version 2.4.6 as follows::
 
-      nrfutil install device=2.4.0 --force
+      nrfutil install device=2.4.6 --force
```

Comment:
See above commane

Reply:
@kenr which comment? Could you be more explicit?

Reply:
The entire document is using nrfutil-device 2.4.6, if we upgrade this one place we need to upgrade here also.
The latest version of nrfutil-device is currently 2.5.0.

Reply:
Ok, clarified with @shanthanordic , the version is 2.4.6